components/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth2/util/OAuth2Util.java

```diff
@@ -2702,6 +2734,7 @@ public static String getFederatedIdPFromDomain(String userStoreDomain) {
      * @param tenantDomain    tenent domain
      * @return an instance of AuthenticatedUser{@link AuthenticatedUser}
      */
+    @Deprecated
```

Comment:
Please add a new method to be used instead

Reply:
Fixed with https://github.com/wso2-extensions/identity-inbound-auth-oauth/pull/1128